packages/expo/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -195,6 +265,21 @@ open class ExpoAppDelegate: ExpoAppInstance {
       }
     }
   }
+#elseif os(macOS)
+  open override func application(
+    _ application: NSApplication,
+    didReceiveRemoteNotification userInfo: [String: Any]
+  ) {
+    let selector = #selector(application(_:didReceiveRemoteNotification:))
+    let subs = ExpoAppDelegateSubscriberRepository.subscribers.filter { $0.responds(to: selector) }
+
+    if !subs.isEmpty {
+      subs.forEach { subscriber in
+        subscriber.application?(application, didReceiveRemoteNotification: userInfo)
+      }
+    }
```

Comment:
```suggestion
    subs.forEach { subscriber in
      subscriber.application?(application, didReceiveRemoteNotification: userInfo)
    }
```

maybe not necessarily to check empty beforehand?